UI/Web/src/app/book-reader/_components/book-reader/book-reader.component.ts

```diff
@@ -996,36 +1071,66 @@ export class BookReaderComponent implements OnInit, AfterViewInit, OnDestroy {
     return this.readingSectionElemRef.nativeElement.clientWidth - margin + columnGap;
   }
 
+  getPageHeight() {
+    if (this.readingSectionElemRef == null) return 0;
+    const height = (parseInt(this.ColumnHeight.replace('px', ''), 10));
+
+    const columnGap = 20;
```

Comment:
Move this and the other into one at the top. We shouldn't have 2 constants in diferent places.